src/app/pages/forms/components/layouts/components/imageCropper/imageCropper.component.ts

```diff
@@ -0,0 +1,73 @@
+import {Component, ViewChild} from '@angular/core'
+import {ImageCropperComponent, CropperSettings, Bounds} from 'ng2-img-cropper';
+
+
+@Component({
+  selector: 'image-cropper',
+  template: `
+    <div>
+     
+     <img-cropper [image]="data1" [settings]="cropperSettings1" (onCrop)="cropped($event)"></img-cropper>
+     <span class="result" *ngIf="data1.image" >
+        <img [src]="data1.image" 
+        [width]="croppedWidth" 
+        [height]="croppedHeight">
+        {{data1.image.length}}
+    </span>
+    
+
+    </div>
+  `,
+})
+export class ImageCropper {
+    name:string;
+    data1:any;
+    cropperSettings1:CropperSettings;
+    croppedWidth:number;
+    croppedHeight:number;
+    
+    @ViewChild('cropper', undefined) cropper:ImageCropperComponent;
```

Comment:
You can use `@ViewChild(ImageCropperComponent)` instead of `@ViewChild('cropper', undefined)`.